ores/applications/util.py

```diff
@@ -0,0 +1,27 @@
+import glob
+import os.path
+import yamlconf
+import logging.config
+
+DEFAULT_DIRS = ["config/", "/etc/ores/"]
+
+
+def build_config(config_dirs=DEFAULT_DIRS,
+                 logging_config="logging_config.yaml"):
+    config_paths = []
+    for directory in config_dirs:
+        config_paths.extend(glob.glob(os.path.join(directory, "*.yaml")))
+    config_paths.sort()
```

Comment:
This would cause tons of issues for prod. you need to sort them in glob.glob(os.path.join(directory, "*.yaml")) and then extend.